api/v1/mongodbcommunity_types.go

```diff
@@ -461,6 +461,13 @@ func (m MongoDBCommunity) NamespacedName() types.NamespacedName {
 	return types.NamespacedName{Name: m.Name, Namespace: m.Namespace}
 }
 
+func (m MongoDBCommunity) GetName() string {
+	return m.Name
+}
+func (m MongoDBCommunity) GetNamespace() string {
+	return m.Namespace
+}
```

Comment:
Removing this makes building fails because

```	"github.com/mongodb/mongodb-kubernetes-operator/api/v1".MongoDBCommunity does not implement statefulSetSpec (GetName method has pointer receiver)```

And I think it's clearer to declare them to compatibility with the AppDB in Enterprise, which doesn't have ObjectMeta. What do you think?

Reply:
ah sure makes sense, I'm okay with it 👍